phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
I think this change should be put on hold untill releasing it in the next major version `2.0.0`, as it is considered a breaking change for those still relying on the old behavior (https://github.com/gruntjs/grunt-contrib-qunit/pull/127#discussion_r63293338).